### PR TITLE
Spanish feedback

### PR DESCRIPTION
**Pull request by sethvargo**
**Release Note**

```release-note
NONE
```

### CODE REVIEW

File: internal/i18n/locales/es/default.po [hunk 1]
```diff
@@ -38,7 +38,7 @@ msgid "nav.event-log"
 msgstr "Registro de eventos"
 
 msgid "nav.signing-keys"
-msgstr "Firma de claves"
+msgstr "Clave de firma"
```
Comment (rafaeltello) [inline internal/i18n/locales/es/default.po:41]: Sorry, I should have said "Claves" in the feedback (plural).

Comment (rafaeltello) [reply internal/i18n/locales/es/default.po:41]: Also, I found a couple of sources that use "Llaves de firmado". Let me check with a couple of industry friends from Mexico to get a sense of which one is more common (both are correct).

Comment (rafaeltello) [reply internal/i18n/locales/es/default.po:41]: Ok, after checking with a couple of Mexican security folks, they tell me "Llaves para firmar" is actually the most widely used term (like in https://www.php.net/manual/es/book.gnupg.php ). Part of the problem is that in Mexico, "clave" (key/code/clef) is often used interchangeably with "contraseña" (password).
That being said, "Claves para firmar" will be understood as well though.

Comment (sethvargo) [reply internal/i18n/locales/es/default.po:41]: Awesome - thanks for the background. I imagine we'll enlist you in the future :)

**Review by whaught — APPROVED** [review]

**Pull request merged**